renovate.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "dependencyDashboard": true,
+  "extends": ["config:base"],
+  "timezone": "Europe/London",
+  "azure-pipelines": {
+    "enabled": true
+  },
+  "vulnerabilityAlerts": {
+    "groupName": "renovate",
+    "dependencyDashboardApproval": false,
+    "minimumReleaseAge": "0d",
+    "rangeStrategy": "update-lockfile",
+    "commitMessageSuffix": "[SECURITY]",
+    "branchTopic": "{{{datasource}}}-{{{depName}}}-vulnerability",
+    "prCreation": "immediate"
+  },
+  "packageRules": [
+    {
+      "matchDatasources": ["npm"],
+      "groupName": "Minor frontend dependencies",
+      "matchUpdateTypes": ["minor", "patch"],
+      "schedule": ["before 8am every weekday", "on the first day of the month"],
+      "enabled": false
+    },
+    {
+      "matchDatasources": ["npm"],
+      "groupName": "Major frontend dependencies",
+      "matchUpdateTypes": ["major"],
+      "schedule": [
+        "before 8am every weekday",
+        "every 3 months on the first day of the month"
+      ],
+      "enabled": false
+    },
+    {
+      "matchDatasources": ["nuget"],
+      "groupName": "Minor backend dependencies",
+      "matchUpdateTypes": ["minor", "patch"],
+      "schedule": ["before 8am every weekday", "on the first day of the month"]
+    },
+    {
+      "matchDatasources": ["nuget"],
+      "groupName": "Major backend dependencies",
+      "matchUpdateTypes": ["major"],
+      "schedule": [
+        "before 8am every weekday",
+        "every 3 months on the first day of the month"
+      ]
+    },
+    {
+      "matchDatasources": ["pypi"],
+      "groupName": "python dependencies"
+    },
+    {
+      "matchDatasources": ["node-version"],
+      "groupName": "node version updates",
+      "schedule": [
+        "before 8am every weekday",
+        "every 3 months on the first day of the month"
+      ]
+    },
```

Comment:
Is this going to suggest node major version updates? I assume we don't want that.

Reply:
I've removed this for now as I believe it will try to upgrade to the latest major which is definitely not what we want. We can always keep an eye on the latest LTS releases for node anyways